lunes_cms/cms/admins/feedback_admin.py

```diff
@@ -23,6 +23,16 @@ class FeedbackAdmin(admin.ModelAdmin):
     sortable_by = ["content_type", "created_date", "read_by"]
     actions = ["mark_as_read", "mark_as_unread"]
 
+    def count_unread_feedback_entries(self):
+        """
+        Add counter for unread feedback entries
+        """
+        counter = 0
+        for item in Feedback.objects.all():
+            if item.get(read_by=None):
+                counter += 1
+        return counter
```

Comment:
```suggestion
        return Feedback.objects.filter(read_by=None).count()
```
This is a bit more complicated than needed - we can just rely on the normal queryset operations for this task.
Have a look at the [QuerySet API reference](https://docs.djangoproject.com/en/3.2/ref/models/querysets/) if you're interested what you can do with them :wink: